### DISCUSSION TITLE
When using nav for the blog plugin, I lose my automatic navigation, is this expected behavior?

### DISCUSSION THREAD

**Discussion by arretx** [squidfunk/mkdocs-material#5776] [Q&A] [answered]
In order for the blog plugin to work, I need to have a nav: section defined, but that wipes out my entire navigation tree, forcing me to manually create it...

...is there a way to keep everything in tact with the blog plugin working?  If not, it's okay, I just don't know if I'm missing something here or not.  Thanks.

**Comment by squidfunk** [comment, marked as answer]
Duplicate of #5132. It's coming, but a little later.

**Reply by arretx** [reply]
Thanks!  I thought I was losing my mind.
